src/server/terrain/simple_structures/SbbGen.zig

```diff
@@ -0,0 +1,93 @@
+const std = @import("std");
+
+const main = @import("main");
+const terrain = main.server.terrain;
+const Vec3i = main.vec.Vec3i;
+const GenerationMode = terrain.biomes.SimpleStructureModel.GenerationMode;
+const CaveMapView = terrain.CaveMap.CaveMapView;
+const CaveBiomeMapView = terrain.CaveBiomeMap.CaveBiomeMapView;
+const sbb = terrain.structure_building_blocks;
+const Blueprint = main.blueprint.Blueprint;
+const ZonElement = main.ZonElement;
+const Neighbor = main.chunk.Neighbor;
+const ServerChunk = main.chunk.ServerChunk;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+const parseBlock = main.blocks.parseBlock;
```

Comment:
unused, also avoid aliasing functions as stated in the contributing guidelines.

Reply:
Ye, sorry, old change